### PR TITLE
Update UUIDs of Services

### PR DESCRIPTION
**Pull request by openBackhaul**
**Process for updating the UUIDs of Services**
- [x] Thorsten to check categories of OperationServers and OperationClients in ServiceList
- [x] Iswaryaa to review categories of OperationServers and OperationClients in ServiceList
- [x] Katharina to add new UUIDs based on category of operation (but not old UUID) in ServiceList
- [x] Iswaryaa to review new UUIDs in ServiceList
- [x] Katharina to CTRL+h for old UUIDs and replace them by new UUIDs in ForwardingList, CONFIGfile and OAS.
- [x] Katharina to step through ForwardingList, CONFIGfile and OAS and compare every UUID against new one in ServiceList
- [ ] Iswaryaa to review new UUIDs in ForwardingList, CONFIGfile and OAS
If no errors found
- [ ] Iswaryaa to delete old UUIDs in ServiceList
- [ ] Iswaryaa to merge the feature branch into develop
- [ ] Iswaryaa to close this issue and delete the feature branch on GitHub

### CODE REVIEW

**Comment by IswaryaaS** [comment]
AdministratorAdministration+services.yaml file has all the services placed under right categories.

**Comment by IswaryaaS** [comment]
Services.yaml file looks fine for new uuid updates. We can proceed updating the other files with new uuid(s)

**Comment by kmohr-soprasteria** [comment]
forwardings list didn't need an update

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: AdministratorAdministration+data.json [hunk 1]
```diff
@@ -1814,17 +1814,17 @@
                   {
```
[the remark this answers is not in the document]
Comment (kmohr-soprasteria) [reply AdministratorAdministration+data.json:1814]: shall not be changed with this pull request

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**